pkg/operation/common/types.go

```diff
@@ -58,20 +58,28 @@ const (
 
 	// ConfirmationDeletion is an annotation on a Shoot resource whose value must be set to "true" in order to
 	// allow deleting the Shoot (if the annotation is not set any DELETE request will be denied).
-	ConfirmationDeletion = "confirmation.garden.sapcloud.io/deletion"
+	ConfirmationDeletion = "confirmation.gardener.cloud/deletion"
```

Comment:
This is also used for confirming a `Project` deletion, right?
Can you adapt the doc string accordinly, please?